apiserver/root.go

```diff
@@ -79,6 +79,7 @@ func newAPIHandler(srv *Server, st *state.State, rpcConn *rpc.Conn, modelUUID st
 		modelUUID:  modelUUID,
 		serverHost: serverHost,
 	}
+
```

Comment:
do we need these?

Reply:
need what? whitespace? accidentally snuck in as I added a resource, then removed it again - reverted